dashboard/src/components/Catalog/Catalog.tsx

```diff
@@ -107,7 +114,8 @@ function Catalog(props: ICatalogProps) {
   useEffect(() => {
     const newFilters = {};
     Object.keys(propsFilter).forEach(filter => {
-      newFilters[filter] = propsFilter[filter]?.toString().split("|");
+      const filterValue = propsFilter[filter]?.toString() || "";
+      newFilters[filter] = filterValue.split(",").map(a => a.replace(/__/g, ","));
```

Comment:
it's usually good to avoid regexps (they are more difficult to compute). You can do `replaceAll("__", ",")`

Reply:
That's was my first option indeed but... tests stop working! The problem is `replaceAll` is not so widely supported, (see https://caniuse.com/mdn-javascript_builtins_string_replaceall). In nodejs, it's supported from >=15, so Jest/Enzyme complains about it's not a function.
So I just replace the replaceAll by a normal replace :P 

Also, I think we should create the regex outside the loop indeed.

Reply:
:O thought that was there from before, nevermind then